docs/whirlpool/docs/05-More Resources/08-Interact via CPI.mdx

```diff
@@ -0,0 +1,28 @@
+import DocCard from '@theme/DocCard';
+
+# Interact via CPI
```

Comment:
We should probably remove this (or add it to Legacy SDK) in favor of `orca_whirlpools_client` from our rust sdk. Ticket?